distribution/scenes/textures/patterns/crackle_solid.pov

```diff
@@ -2,97 +2,190 @@
 // To view a copy of this license, visit http://creativecommons.org/licenses/by/3.0/
 // or send a letter to Creative Commons, 444 Castro Street, Suite 900, Mountain View,
 // California, 94041, USA.
-
-// Persistence Of Vision raytracer sample file.
-// This file demonstrates some uses of the "crackle solid" pattern,
-// with a fake-granite tiled floor, and a zinc bucket.
 //
-// -w320 -h240
+// Persistence Of Vision Ray Tracer ('POV-Ray') sample file.
+//
+// Crackle pattern example crackle_solid.pov.
+//
 // -w800 -h600 +a0.3
 
-#version 3.7;
-
+#version 3.71;
 global_settings { assumed_gamma 1.0 }
+#default { finish { ambient 0.006 diffuse 0.456 } }
 
-#include "colors.inc"
-#include "woods.inc"
-
-camera { location <70,100,60> 
-         right   x*image_width/image_height
-         angle 40 
-         look_at <0,15,0>
-       }
+camera {
+    location <70,100,60>
+    right    x*image_width/image_height
+    angle    40
+    look_at  <0,15,0>
+}
 
+#declare White = srgb <1,1,1>;
 light_source {<40,350,300> White*1.5 spotlight point_at 0 radius 9 falloff 11}
 
 //--textures definitions--------------------------------
-
+#declare Black  = srgb <0,0,0>;
+#declare Gray60 = srgb <0.6,0.6,0.6>;
+#declare Gray80 = srgb <0.8,0.8,0.8>;
+#declare Gray90 = srgb <0.9,0.9,0.9>;
+#declare Red    = srgb <1,0,0>;
+#declare Tan    = srgb <0.86646,0.58571,0.44532>;
+#declare Wheat  = srgb <0.85494,0.85494,0.75836>;
```

Comment:
Perhaps some of this is my experience. Remembering back to my start with POV-Ray I found the includes for color.inc NOT useful. The colors beyond a very few did not correspond to anything I used. Today I look at these and wonder their source - not to mention the SDL encoding is odd when new users go looking for colors.inc. 

I know you encouraged the colors.inc overhaul & I looked at it and other includes, but dang this ripples into everything. I'm just not ready to dive into that amount of grunt work and inevitable inevitable complaints some scene or other doesn't look right. 

In one scene's changes (tiling.pov?) I made use of SVG1.0 standard colors which are almost identical to X11 and .net standards for the web. I have the full set encoded in my working space & have been playing with it. My thinking is maybe we decouple the old POV-Ray colors in colors.inc from anything 3.7 and newer by going with the SVG1.0 set for anything updated, but not convinced of this path myself yet. 

What I decided to do for now is update and decouple from 'gamma' tangled includes for now. We can more quickly get to a set of scenes for new users that are not 'washed out' in appearance.  Such a  'step' doesn't preclude further updates.